theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-opened-traces-widget.tsx

```diff
@@ -223,6 +259,9 @@ export class TraceExplorerOpenedTracesWidget extends ReactWidget {
 
     protected async doUpdateOpenedExperiments(): Promise<void> {
         this._openedExperiments = await this.experimentManager.getOpenedExperiments();
+        this._openedExperiments.forEach(experiment => {
+            this.experimentManager.addExperiment(experiment);
```

Comment:
Open a trace, close the trace, close the browser. Restart the trace-server, then reopen the browser.

This code will add the created-but-closed experiment to the experiment-manager. But it does not add its traces to the trace-manager. So if you try to delete this experiment, its traces won't be be deleted on the trace-server because the trace-manager doesn't have them in its open traces map.